src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/event/PrisonOffenderEventListener.kt

```diff
@@ -32,7 +30,6 @@ class PrisonOffenderEventListener(
   }
 
   @SqsListener("domaineventsqueue", factory = "hmppsQueueContainerFactoryProxy")
-  @WithSpan(value = "hmpps-assess-for-early-release-prisoner-domain-event-queue", kind = SpanKind.SERVER)
```

Comment:
these spans are auto registered by the sqs library now so need to remove these to prevent double logging